about/policy/sgid/index.md

```diff
@@ -1,116 +1,171 @@
+### Dates in Table Names
 {: .text-left }
 
-Over time, we may remove datasets from the SGID geodatabase for various reasons, some of which include the following:
+Dates are acceptable in table names when content belongs in either the `census`, `political`, or `tax` categories to identify datasets where the most current data may not reflect the current calendar year.
 
-- We have a new version of the dataset with a substantially different schema.
-- The dataset is out-of-date.
-- The dataset is no longer being updated by the steward.
-- The steward has requested us to remove the dataset.
-- A more authoritative source for the dataset exists elsewhere.
-- The dataset does not meet the qualifying criteria listed in the "Qualifying Datasets" section above.
+For all other categories, dates should only be used for tables that are no longer effective/active or are soon-to-be effective/active. This implies that the **most current and relevant** data contains **no date suffix**. Historical or future data should contain a date suffix to help identification at a glance.
 
-Just like when adding data, we will use the [Porter](https://github.com/agrc/porter) project to track and guide dataset deprecation.
+The best way to learn about a dataset is to view the data page metadata.
 
-We acknowledge the impact removing datasets can have on maps and projects using our datasets. We will do our best to communicate and warn people as quickly as possible through our [@MapUtah](https://twitter.com/maputah) Twitter account and by marking items as `Deprecated` in ArcGIS Online before deleting them. We encourage everyone using our data to follow us on Twitter.
+### Internal Database Standards
+{: .text-left}
 
-### "Static" Datasets
-{: .text-left }
+The SGID uses the ESRI default Resolution and Tolerance:
 
-We anticipate some SGID datasets won't receive future updates, such as the boundary lines for adjacent counties in the surrounding states. These datasets will be removed from the SGID geodatabase but left in ArcGIS Online, where they will be tagged as `static` and remain members of a Utah SGID group.
+- XY Resolution: 0.0001 Meter
+- XY Tolerance: 0.001 Meter
 
-### "Shelved" Datasets
-{: .text-left }
+Any coded-value domain codes must match their values (for example, `"Interstate Highway" - "Interstate Highway"` instead of `1 - "Interstate Highway"`).
+
+### Editing Policies
+{: .text-left}
+
+No edits or updates should be made to the internal geodatabase from 7:00PM to 10:00AM MST to allow automated scripts to push dataset updates to ArcGIS Online and the Open SGID. AGRC should ensure that data is production-ready before the editing window ends.
+
+Whenever possible, internal geodatabase edits and updates should be done using auotmated tools that move data from a secondary workspace into the internal geodatabase to reduce the time of incomplete data in the internal geodatabase. (Is this neccessary now that we're the only ones hitting internal SGID?)
```

Comment:
I don't think that this is a big deal anymore as long as the timeframe from the paragraph above is honored.